composer.json

```diff
@@ -16,15 +16,19 @@
         {
             "name": "Rodney Rehm",
             "email": "mail+github@rodneyrehm.de"
-        }
-    ],
+        },
+        {
+         "name": "Thierry Bugier",
+         "email": "tbugier@teclib.com"
+     }
+ ],
     "support": {
```

Comment:
"irc": "irc://irc.freenode.org/flyve-mdm",
"email": "contact@teclib.com", 
"docs": "http://teclib.github.io/CFPropertyList/",
"rss": "https://teclib.github.io/CFPropertyList/feed.xml"

Reply:
done ,except irc, as we discussed